### PR TITLE
Remove the word 'CAS' from SBOM document

### PR DESCRIPTION
**Pull request by KAWAHARA-souta**
CAS is not actually used anymore. It needs to be removed.

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -164,12 +164,12 @@
    "How are we doing this?": "How are we doing this?",
    "AlmaLinux is leveraging Codenotary’s open source": "AlmaLinux is leveraging Codenotary’s open source",
    "to provide administrators with authentication, verification and full SBOM visibility.": "to provide administrators with authentication, verification and full SBOM visibility.",
-   "CAS stores all signatures inside of": "CAS stores all signatures inside of",
+   "Build System stores all signatures inside of immudb": "Build System stores all signatures inside of immudb",
```
Comment (javihernandez) [inline i18n/en.json:167]: I'd rather say "The AlmaLinux Build System stores SBOM data inside of immudb ...."

Comment (KAWAHARA-souta) [reply i18n/en.json:167]: Thank you for reviewing!I have a question.

My understanding is following.
"The data stored inside of immudb is nortarized codes and/or packages data (=signatures), and alma-sbom.py makes SBOM data from that."
Is this possibly incorrect...?

Comment (javihernandez) [reply i18n/en.json:167]: The data we store in immudb is mostly, notarized records of git sources and files. The data that these immudb records include is described in detail [here](https://github.com/AlmaLinux/build-system-rfes/blob/master/SBOM/SBOM.md). 
And yes, this is the data that alma-sbom uses to create SBOM records.

Comment (KAWAHARA-souta) [reply i18n/en.json:167]: I got it, thanks!
I'll apply your comment soon.

**Review by javihernandez — APPROVED** [review]

**Review by bennyvasquez — APPROVED** [review]
Thanks, @KAWAHARA-souta and @javihernandez! This is great.

**Pull request merged**